src/helper/securityQuestion.ts

```diff
@@ -0,0 +1,152 @@
+import { decrypt, encrypt, EncryptedMessage, getPubKeyECC, StringifiedType } from "@tkey-mpc/common-types";
+import { keccak256 } from "@toruslabs/torus.js";
+import BN from "bn.js";
+
+import { FactorKeyTypeShareDescription, TssFactorIndexType } from "../constants";
+import { Web3AuthMPCCoreKit } from "../mpcCoreKit";
+
+export class TssSecurityQuestionStore {
```

Comment:
i think no need to export this store class, its better if app doesn't touches it.